tasks/packagemgmt.yml

```diff
@@ -205,9 +205,11 @@
 
 - name: dnf upgrade
```

Comment:
yeah, I want to do this as well and done so in the paste, but it triggers `package-latest Package installs should not use latest` (see https://github.com/ansible-community/ansible-lint/issues/479).
The irony is of course that https://github.com/konstruktoid/ansible-role-hardening/blob/master/tasks/packagemgmt.yml#L36-L42 doesn't trigger this rule.

Exclude this from `ansible-lint` (https://ansible-lint.readthedocs.io/en/latest/rules.html#false-positives-skipping-rules) and we'll be fine.

Reply:
I'm not sure if I catch everything correctly here, butI have added the 

`# noqa 403`

to the rule. If it's not enough, please give me more exact suggestion how to make the taks compliant with this requirement.

Reply:
No, that would do it